modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -104,30 +113,357 @@ Execution Type::
+
+Use the dashboard to compare cores and flows consumed by each application side-by-side to help you understand your utilization and associated pricing. You can visualize cores and flows consumed by each application alongside each other and drill down per environment ID and business group when you export the usage report CSV file. 
+
+If you are using the cluster capacity metric for billing, this translation from core usage to flow usage enables you to make the necessary changes to applications so you can transition to usage-based pricing iteratively, which gives you a more flexible pricing model that scales with your integration needs and isn't tied to your infrastructure.
+
+=== Runtime Fabric Agent Version and Configuration Requirements
+
+There are some additional configuration requirements if you are using the cluster capacity metric for tracking. These requirements don't apply for accurate tracking of the CPU limit metric. 
+
+* The Runtime Fabric agent must be version 2.7.0 or later. 
++
+For more information, see xref:runtime-fabric::upgrade-index.adoc[].
+* If you are using a Helm-based Runtime Fabric installation, ensure that the NODE_WATCHER parameter is enabled.
++
+For more information about enabling NODE_WATCHER, see xref:runtime-fabric::install-helm.adoc#optional-parameters[Optional Parameters].
+
+These configurations only apply to the Runtime Fabric agent and have no impact on applications. You don't need to take any action on applications (redeployment or restarts) for tracking the cluster capacity and CPU limit metrics.
+
+[[rtf-compliance-tracking]]
+=== Runtime Fabric Compliance Tracking With Usage Metrics
+
+You must adjust your clusters and application configurations for accurate measurement of usage metrics. You can continue to choose whether to use the CPU limit metric or cluster capacity metric, at the root organization level for contract governance depending on your use case. 
+
+Here are some recommendations to help you accurately measure metrics:
+
+[%header%autowidth.spread]
+|===
+| CPU Limit | Cluster Capacity
+a|* Each application requires variable CPU allocation depending on its workload requirements. Use Anypoint Monitoring to analyze the CPU usage of your application during development to identify the right CPU limit and CPU reserve configurations. 
+* Runtime Manager sets the CPU limit configuration to the available cluster capacity of the underlying Runtime Fabric instance by default, which can result in a higher CPU limit usage reported for applications if the CPU limit configuration isn't explicitly modified during deployment. For an accurate CPU Limit metric, ensure that the application's CPU Limit configurations are updated with the appropriate values based on the app's performance.
```

Comment:
```suggestion
* Runtime Manager sets the CPU limit configuration to the available cluster capacity of the underlying Runtime Fabric instance by default. This setting can result in a higher CPU limit usage reported for applications if the CPU limit configuration isn't explicitly modified during deployment. For an accurate CPU limit metric, verify that the application's CPU limit configurations are updated with the appropriate values based on the app's performance.
```

Reply:
I think, unless you're referring to an actual UI element (in bold), you can lc these.